docker/pilot_ingest.py

```
@@ -0,0 +1,76 @@
+
+from datetime import datetime
+import sys
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+try:
+    MAX_ISSUES = int(os.getenv("MAX_ISSUES", "200"))
+    if MAX_ISSUES < 1:
+        raise ValueError("MAX_ISSUES must be at least 1")
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)